### ISSUE TITLE
General page in Administrative Area does not load form data after upgrading

### ISSUE THREAD

**Issue by fireundubh** [Requarks/wiki#1510]
After upgrading from 2.0.12 to 2.1.113, the General page in the Administrative Area no longer loads any data into the form.

An error notification appears overhead: `An unexpected error occured.`

When I enter data into the now-empty Site URL and Site Title fields and click Apply, I receive the following error notification: `Unknown argument "hasLogo" on field "updateConfig" of type "SiteMutation".`

Despite the blank fields, my wiki is still online at its previously set Site URL and retains its Site Title from 2.0.12.

**Comment by fireundubh** [comment]
I entered my wiki name into the Site Title and clicked Apply. I received the following error notification: Unknown argument "hasLogo" on field "updateConfig" of type "SiteMutation".

I loaded up the database in SQLyog and noted that only one or two of the rows had a recent updatedBy value.

I navigated away, clicked a few more pages, and clicked back to General. No error notification and the form data loaded. Now, half of the rows have a recent updatedBy value.

The issue seems to have resolved itself. 🤷‍♂️
